mail_outplacement_report/__manifest__.py

```diff
@@ -2,7 +2,7 @@
 
 {
     "name": "DAFA Outplacement Templates",
-    "version": "12.0.2.0.0",
+    "version": "V12.0.1.0.2",
```

Comment:
There shouldn't be a V in front of the version number, and the number isn't supposed to move backwards.